src/components/ThreejsEditorModal.js

```diff
@@ -35,11 +37,17 @@ export class ThreejsEditorModal extends ModalDialog {
      */
     initializeEditor() {
 
-        this.editor = new window.Editor();
+        THREE.Object3D.DefaultUp.set(0, 0, 1);
+
+        const camera = new THREE.PerspectiveCamera(50, 1, 0.01, 20000);
```

Comment:
Let's explain the logic in an inline comment pls

Reply:
Added.